deploy/config/keter.yaml

```diff
@@ -1,8 +1,9 @@
 stanzas:
-  - type: webapp
+  - type: background
```

Comment:
I'm just curious, but why does this need to be `background` instead of `webapp`?

I would have assumed that `webapp` was correct, so I'm just surprised it's been changed to `background`.

To be honest, I don't really know the difference between `webapp` and `background` for keter.